Code/Framework/AzToolsFramework/AzToolsFramework/UI/DPEDebugViewer/DPEDebugModel.cpp

```diff
@@ -347,14 +473,8 @@ namespace AzToolsFramework
 
     int DPEDebugModel::columnCount(const QModelIndex& parent) const
     {
-        int columnCount = 0;
-
-        DPEModelNode* theNode = (parent.isValid() ? GetNodeFromIndex(parent) : m_rootNode);
-        if (theNode)
-        {
-            columnCount = theNode->GetMaxChildColumns();
-        }
-        return columnCount;
+        Q_UNUSED(parent);
```

Comment:
nit: prefer [[maybe_unused]]

Reply:
nit taken, though I am deeply in Qt land for the entirety of this module :P
fixed now.